File: r-package/examples/brainflow_get_data.R
library(brainflow)

board_shim <- get_board_shim(Boards()$Cython["Type"], "/dev/ttyUSB0")
prepare_session(board_shim)
start_stream(board_shim, 3600)
Sys.sleep(time = 5)
stop_stream(board_shim)
data <- get_current_board_data(board_shim, 250)
release_session(board_shim)

data_handler <- get_data_handler(Boards()$Cython["Type"], numpy_data = data)
preprocess_data(data_handler, 1, 50, TRUE)
preprocessed_data <- get_data(data_handler)
head(preprocessed_data)

File: matlab-package/BrainflowGetData.m
board_shim = BoardShim (BoardIDs.GANGLION_BOARD, '');
board_shim.prepare_session ();
board_shim.start_stream (3600);
pause (5)
board_shim.stop_stream ()
[data, ts] = board_shim.get_current_board_data (250);
disp (data);
board_shim.release_session ()